client/src/views/SolutionPage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="mj-solution-section">
+    <div v-if="mistake">
+      <div class="mj-solution-section-titles">{{ $t('Solutions.Name') }}</div>
```

Comment:
I think the class should be named `mj-solution-section-title` instead of `mj-solution-section-titles` :)

Reply:
done :)